.github/workflows/node.js.yml

```diff
@@ -10,31 +7,48 @@ on:
   pull_request:
 
 jobs:
-  build:
```

Comment:
Renamed this one to `test` as there is no build step, and the only purpose of this job is to run the tests.